pkgs/development/tools/sublime-merge/common.nix

```diff
@@ -0,0 +1,87 @@
+{buildVersion, x64sha256}:
+
+{ fetchurl, stdenv, glib, xorg, cairo, gtk2, pango, makeWrapper, openssl, bzip2,
+  pkexecPath ? "/run/wrappers/bin/pkexec", libredirect,
+  gksuSupport ? false, gksu, unzip, zip, bash}:
+
+assert gksuSupport -> gksu != null;
+
+let
+
+  libPath = stdenv.lib.makeLibraryPath [glib xorg.libX11 gtk2 cairo pango];
+  redirects = [ "/usr/bin/pkexec=${pkexecPath}" ]
+    ++ stdenv.lib.optional gksuSupport "/usr/bin/gksudo=${gksu}/bin/gksudo";
```

Comment:
Maybe we can convince upstream to not use absolute paths here.

Reply:
Is the sublime forum the right place to ask? @guillermooo

Reply:
I dropped something here, let's see: https://forum.sublimetext.com/t/sublime-tries-absolute-paths-for-pkexec-gksudo/39362